### PR TITLE
Upgrade JDK17 jenkins jobs references

### PR DESCRIPTION
**Pull request by pjgg**
Upgrade Quarkus-728 test plan according new Jenkins job spec

### CODE REVIEW

File: QUARKUS-728.md [hunk 1]
```diff
@@ -25,35 +24,13 @@ Test development will focus on
    - OpenShift OSD with UBI8 OpenJDK 17
    - RHEL 9 x86_64 with OpenJDK 17
   
- That could be reflected in the following Jenkins jobs:
+ That could be reflected in the following Jenkins matrix jobs:
```
Comment (mjurc) [inline QUARKUS-728.md:27]: Think the "matrix" part could be omitted - one of the jobs is not matrix, after all :)

**Review by mjurc — CHANGES_REQUESTED** [review]

**Review by rsvoboda — CHANGES_REQUESTED** [review]
remove `.idea/.xml files`

**Review by mjurc — APPROVED** [review]
LGTM, thanks for the update

**Comment by rsvoboda** [comment]
@mjurc you merged unwanted .idea files :/

**Pull request merged**